helpers/FOLDER.py

```diff
@@ -51,15 +51,14 @@ def get_job_dict(tiles_gdf, base_path, end_path='all-images', year=None, save_me
             'basepath': base_path,
             'filename': image_path,
             'bbox': bbox,
-            'year': tile.year_tile if 'year_tile' in tiles_gdf.keys() and str(year).isnumeric()==False else year,
             'save_metadata': save_metadata,
             'overwrite': overwrite
         }
 
     return job_dict
 
 
-def get_image_to_folder(basepath, filename, bbox, year, save_metadata=False, overwrite=True):
```

Comment:
It is needed if the key 'year' is present in `job_dict` otherwise it raises an error:
![image](https://github.com/user-attachments/assets/476da726-a449-4b64-8062-0570034da629)
The year is not explicitly used in the `get_image_to_folder` function because all the .tif file are read from the given folder. The year info is needed later. If it's just to make sonar cube happy, we can add the year to the image_metadata which would make sense anyway or find another way to do it.

Reply:
Isn't the mistake because I forgot to remove the year at line 553 of `generate_tilesets.py`. Would it not be ok if I just remove it?